LiftOff-Project/ClientApp/.env

```diff
@@ -1 +1,3 @@
 BROWSER=none
+API_KEY = "?api_key=627127f14caca839ef42d22a23dcefde";
```

Comment:
This file should be in gitignore, but we can all copy it on this one and then add it to gitignore